src/com/owncloud/android/ui/fragment/FileFragment.java

```diff
@@ -144,7 +144,7 @@ public void onSyncEvent(String syncEvent, boolean success, OCFile updatedFile) {
          * 
          * @param file      File to show details
          */
-        public void showDetails(OCFile file);
+        void showDetails(OCFile file);
 
         
```

Comment:
Please update the copyright year to 2018 and include yourself as @author at the top of the class

Reply:
done